src/components/four-range-control/index.js

```diff
@@ -133,6 +148,45 @@ const FourRangeControl = memo( props => {
 		propsToPass.placeholder = props.placeholder
 	}
 
+	const setInitPositionPlaceholder = ( _unit, _value, setAll ) => {
+		propsToPass.initialPositionTop = unit === _unit ? _value?.top : ''
+		propsToPass.placeholderTop = unit === _unit ? _value?.top : ''
+		propsToPass.initialPositionLeft = unit === _unit ? _value?.left : ''
+		propsToPass.placeholderLeft = unit === _unit ? _value?.left : ''
+		if ( setAll ) {
+			propsToPass.initialPositionRight = unit === _unit ? _value?.right : ''
+			propsToPass.placeholderRight = unit === _unit ? _value?.right : ''
+			propsToPass.initialPositionBottom = unit === _unit ? _value?.bottom : ''
+			propsToPass.placeholderBottom = unit === _unit ? _value?.bottom : ''
+		}
```

Comment:
`setAll` looks wrong.. because there are various props that can make the four range control show different sets of range controls.